src/__tests__/postgres/integration.test.ts

```diff
@@ -394,4 +394,62 @@ describeMaybeSkip('integration (postgres)', () => {
     expect(errors).toBeUndefined()
     expect(data?.films.some((f: any) => f.sequel && f.sequel.id)).toBe(true)
   })
+
+  test('subscriptions', async () => {
+    const document = parse(`
+      subscription {
+        create 
+      }
+    `)
+
+    const sub = <AsyncIterator<ExecutionResult>>await subscribe(schema, document)
+    expect(sub.next).toBeDefined()
+
+    const next = sub.next() // grab the promise
+    pubSub.publish('CREATE_ONE', { create: 'FLUM!' }) //publish
+    const {
+      value: { errors, data },
+    } = await next // await the promise
+
+    expect(errors).toBeUndefined()
+    expect(data).toBeDefined()
+    expect(data.create).toBe('FLUM!')
+  })
+
+  // this skipped test is returning "GraphQLError: Cannot return null for non-nullable field Subscription.create" at `expect(subErrors).toBeUndefined()`
+  test('subscription triggered by mutation', async () => {
```

Comment:
suggestion: This probably needs to be converted to a unit test for now.

Like I said [before](https://github.com/danielrearden/sqlmancer/pull/121#discussion_r446514166), we should probably start with unit tests to directly exercise any methods that are being added. The unit tests include a harness that lets you roll back any changes when doing mutations so that the tests don't have to worry about side-effects. We should probably add something similar to the integration tests. Right now I don't have any mutations included in the integration tests for this reason.